internal/mode/static/state/conditions/conditions.go

```diff
@@ -271,6 +275,15 @@ func NewRouteGatewayNotProgrammed(msg string) conditions.Condition {
 	}
 }
 
+func NewRouteInvalidIPFamily(msg string) conditions.Condition {
+	return conditions.Condition{
+		Type:    string(v1.RouteConditionAccepted),
```

Comment:
> @kate-osborn @pleshakov I am not sure if the conditions set [here](https://github.com/nginxinc/nginx-gateway-fabric/blob/1481d4df9a00571c4292545a1cf27cdafab70013/internal/mode/static/state/graph/backend_refs.go#L173). Should it references not resolved, i did something to similar to other conditions defined.

we should use this condition type https://github.com/nginxinc/nginx-gateway-fabric/blob/1481d4df9a00571c4292545a1cf27cdafab70013/internal/mode/static/state/conditions/conditions.go#L227 , so that it doesn't affect the route accepted condition